sky/templates/aws-ray.yml.j2

```diff
@@ -47,10 +47,8 @@ available_node_types:
         - DeviceName: /dev/sda1
           Ebs:
             VolumeSize: {{disk_size}}
-            # use default Iops for gp3
-            VolumeType: gp3
-            Iops: 3000
-            Throughput: 125 
```

Comment:
Why do we remove this throughput? It seems the AWS allows to specify the Iops and Throughput separately. Should we consider both in our disk type separation?

Reply:
When I try provision for AWS, I found that some disk type does not support specifying throughput. Also, throughput will be auto-specified from IOPS, so I remove it.

Reply:
Can we explain why the throughput is added back again?

Reply:
Because now I only enable it when disk_type == gp3. Please refer to the `custom_disk_perf` flag.